planetiler-core/pom.xml

```diff
@@ -65,6 +65,16 @@
       <artifactId>gt-epsg-hsql</artifactId>
       <version>${geotools.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.geotools</groupId>
+      <artifactId>gt-geojson</artifactId>
+      <version>${geotools.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.geotools</groupId>
+      <artifactId>gt-geojson-store</artifactId>
+      <version>${geotools.version}</version>
+    </dependency>
```

Comment:
I see JTS already has a [GeoJsonReader](https://github.com/locationtech/jts/blob/master/modules/io/common/src/main/java/org/locationtech/jts/io/geojson/GeoJsonReader.java) class. Geotools has some licensing issues so I'm trying to avoid adding new dependencies on it.  But the JTS reader doesn't seem to handle properties at all so this might be the best one to start with...

Reply:
Yes, JTS is only for geometries, it doesn't handle feature and feature collection...
Geotools uses it under the hood...

Reply:
Does the geotools version automatically handle parsing/converting CRS of input geojson?

Reply:
No clue, probably not, it is very similar to shapefile, I belive, in terms of API.

Reply:
I have tried to play with the Crs object of the store in various way but it seems to always return WGS84 even if the file said otherwise.
I've opened a ticket here:
https://osgeo-org.atlassian.net/browse/GEOT-7705
This can be read from the file itself, but it seems hacky...

Reply:
Ok thanks! The jts version handles crs but not properties, and geotools version handles properties but not crs 🤦‍♂️ if there's just one crs defined for the entire object it wouldn't be too bad to read it out like jts does, but we could also merge this version then handle crs and newline-delimited in a follow-up pr?

Reply:
Sure, I think this is a good addition as is. We can always improve in the future.